azure-pipelines/ui-automation/wpj-test.yml

```diff
@@ -0,0 +1,175 @@
+# run UI automation testcases for wpj
```

Comment:
So this is a completely separate pipeline specifically for WPJ test cases?

Reply:
Yes, at the time this pipeline was created the broker-release pipeline had not to be created.
I created a [PBI](https://identitydivision.visualstudio.com/Engineering/_workitems/edit/1651056) to integrate the wpj pipeline into the broker-release pipeline once this pipeline gets completed
The broker-release pipeline is [still in progress](https://github.com/AzureAD/android-complete/compare/amgusain/broker-ci) Amit picked up the current work and its adding extra functionality.

Reply:
Now that broker-release pipeline is up, should that be used run these tests,